assembl/static2/js/app/components/form/fileUploaderFieldAdapter.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from 'react';
+import { type FieldRenderProps } from 'react-final-form';
+import { HelpBlock, FormGroup } from 'react-bootstrap';
+
+import FileUploader from '../common/fileUploader';
+import { getValidationState } from './utils';
+
+type Props = {
+  label: string
+} & FieldRenderProps;
+
+const FileUploaderFieldAdapter = ({ input, label, meta: { error, touched } }: Props) => {
+  const onChange = (value) => {
+    input.onChange({
+      externalUrl: value,
+      imgTitle: value.name,
+      mimeType: value.type
+    });
+  };
+  return (
+    <FormGroup validationState={getValidationState(error, touched)}>
+      <label htmlFor={input.name}>{label}</label>
+      <FileUploader
+        key={input.value ? 'notEmpty' : 'empty'}
+        fileOrUrl={input.value.externalUrl}
```

Comment:
Are we sure `input.value` is not null here? Apparently not if you have a check for `key`